wifistats/airport.js

```diff
@@ -0,0 +1,40 @@
+const cmdLine = '/System/Library/PrivateFrameworks/Apple80211.framework/Versions/Current/Resources/airport -I';
+
+/**
+ * Parses the output of the {@link cmdLine}.
+ * We search for agrCtlRSSI and agrCtlNoise values.
+ * The result is SNR the difference in decibels between the received signal
+ * and the background noise level.
+ *
+ * @param {string} str - the string which is output of the command.
+ * @param callback
+ */
+function parseOutput(str, callback) {
```

Comment:
this function doesn't do any asynchronous work, why not just return the value or null?

Reply:
The asynchronous was added now :)